packages/ui-stencil/src/styles/_colors.scss

```diff
@@ -71,37 +72,33 @@ $semanticColors: (
 
 $elementColors: (
   'button-text': (
-    // TODO: example of how to use the color functions with semantic colors
-    // primary: color.adjust(text-color('primary', $semanticColors), $lightness: 10%),
-    primary: $purple100,
+    primary: icon-color('reverse', $semanticColors),
     secondary: text-color('secondary', $semanticColors),
     inactive: $gray500,
```

Comment:
@aileenvl can you please check this token? If we're not using it we can remove it, otherwise we have to replace the primitive with the semantic. Ty